analytics_queries/participants.sql

```diff
@@ -36,8 +36,17 @@
             JOIN participant_identities pi on pp.participant_identity_id = pi.id
             LEFT OUTER JOIN ecf_participant_validation_data epvd on pp.id = epvd.participant_profile_id
             LEFT OUTER JOIN ecf_participant_eligibilities epe on pp.id = epe.participant_profile_id
+            JOIN (
+                 SELECT
+                    DISTINCT ON (participant_profile_id) participant_profile_id,
+                    training_status
+                 FROM induction_records
+                 JOIN induction_programmes ON induction_programmes.id = induction_records.induction_programme_id
+                 JOIN partnerships ON partnerships.id = induction_programmes.partnership_id
+                 ORDER BY participant_profile_id, induction_records.created_at DESC
+            ) AS latest_induction_record ON latest_induction_record.participant_profile_id = pp.id
```

Comment:
I think we might need to define that is is one record here, there are examples of this in the `participant_query` and the `ecf_duplicates_query`

Reply:
`ROW_NUMBER() OVER (PARTITION BY participant_profile_id ORDER BY induction_records.created_at DESC) AS induction_record_sort_order` then adding `AND induction_record_sort_order = 1`

Reply:
Hi @cwrw I've just made some changes to keep it consistent with the latest induction records selection from the examples. Tksss